deployment/charts/templates/deploy.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.app.prometheus.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "fuel-block-committer.name" . }}-service-monitor
+  labels:
+    release: {{ .Values.app.prometheus.helm_release_name }}
+spec:
+  selector:
+    app: {{ template "fuel-block-committer.name" . }}
+  endpoints:
+    - path: /metrics
+      port: http
+{{- end }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fuel-block-committer.name" . }}
+    chart: {{ template "fuel-block-committer.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "fuel-block-committer.name" . }}-service
+spec:
+  type: NodePort
+  selector:
+    app: {{ template "fuel-block-committer.name" . }}
+  ports:
+    - name: http
+      port: 80
+      protocol: TCP
+      targetPort: 8080
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "fuel-block-committer.name" . }}-k8s
```

Comment:
Please test this by doing 2 deployments in the same namespace. My feeling is that the second one is not going to work because it has the same object name

Reply:
update to incorporate the chart release name